.github/workflows/ci.yml

```diff
@@ -72,3 +74,14 @@ jobs:
         user: __token__
         password: ${{ secrets.pypi_password }}
       if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+
+  pass:
+    if: always()
```

Comment:
This seems redundant, but maybe it's there for emphasis...

Reply:
I see: there are different notions of "always." This isn't just a predicate that evaluates to true; it's overriding the usual situation in which a workflow will stop on first failure.